rules/follow-naming-conventions-for-tests-and-test-projects/rule.md

```diff
@@ -61,21 +63,29 @@ Remember that descriptive names are useful – but the choice of name is not the
 :::
 
 ```cs
-[Method/PropertyName]_Should_[ExpectedBehavior]_When_[StateUnderTest]
+{SUT}_{Condition}_{Should}
 ```
+**Figure: The naming convention is effective – it encourages developers to clearly define the expected result upfront without requiring too much verbosity** 
 
-**Figure: The "should" naming convention is effective – it encourages developers to clearly define the expected result upfront without requiring too much verbosity** 
+Note: 'SUT' is a System Under Test.
 
-The following test names use the "should" naming convention:
+Think of this as 3 parts, separated by underscores:
+- WHEN (SUT, e.g. Checkout)
+- GIVEN (Condition, e.g. Country is Australia)
+- THEN (Should, e.g. Add 10% tax)
+
+The following test names use the naming convention:
 
 ```cs
-Withdraw_Should_ThrowException_When_InvalidAccountInfo
-Checkout_Should_Add10Percent_When_CountryIsAustralia 
-Purchase_Should_Succeed_When_BalanceIsWithinCreditLimit
+Withdraw_WithInvalidAccount_ThrowsException
+Checkout_WithCountryAsAustralia_ShouldAdd10PercentTax
+Purchase_WithBalanceWithinCreditLimit_ShouldSucceed
 ```
```

Comment:
For GWT, I'd suggest:

`GivenInvalidAccount_WhenCheckout_ThenThrowsException`
`GivenCountryAsAustralia_WhenCheckout_ThenAdd10PercentTax`
`GivenBalanceWithinCreditLimit_WhenCheckout_ThenReturnSuccess`